README.md

```diff
@@ -1,20 +1,23 @@
 # Heroku Buildpack - Voice
 
-Simple Heroku Buildpack to setup a Nexmo Voice application, purchse a new Phone Number and Link that number to the applicaiton.
+This is a simple Heroku Buildpack to setup a Nexmo Voice application. Purchase a new phone number and link that number to the application.
```

Comment:
The buildpack does all of the above. putting a stop in the middle implies you have to do the latter.